picdar-export/src/main/scala/com.gu.mediaservice.picdarexport/lib/Config.scala

```diff
@@ -39,8 +39,8 @@ object Config {
   val concurrencyMediaApi    = properties("concurrency.media.api").toInt
 
   // Configure timeouts for all services
-  private val defaultConnectionTimeout =  5000 // ms
-  private val defaultReadTimeout       = 15000 // ms
+  private val defaultConnectionTimeout = 35000 // ms
```

Comment:
We can read much larger volumes of data from `fakepicdar` so allow more time to send it over.